test/functional/DDR_Test/tck_ddrext.xml

```diff
@@ -147,10 +151,13 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 			<arg line="${DUMP_OPTION}" />
 			<arg line="-Xitsn1000" />
 			<arg line="${EXTRADUMPOPT}" />
-			<arg value="-Xjit:{j9vm/test/corehelper/CoreGen.main*}(count=0)"/>
+			<arg line="${JVM_OPTIONS}" />
+			<arg line="-Djava.library.path=${LIBPATH}"/>				
```

Comment:
Only certain tests have LIBPATH set. This change will set  java.library.path  to empty for other tests. Please a variable as arg, similar to `${JVM_OPTIONS}`.